src/fluree/db/query/exec/eval.cljc

```diff
@@ -932,9 +992,12 @@
                                     'format #?(:clj format :cljs str))
```

Comment:
Will these actually work the same way? I suppose this isn't used in `:cljs` yet, but I think a TODO might be useful here.

Also, I think you need to us the `syntax-quote` backtick here instead of the `quote` apostrophe if you want `format` to be namespace qualified.